### ISSUE TITLE
[BUG] Running "viash" with no options throws an error

### ISSUE THREAD

**Issue by mberacochea** [viash-io/viash#617]
### What happened?

I get the following error when running viash with no options

### Steps to reproduce

Download and install viash, and run `viash` with no options or flags

### Expected behavior

It should probably print the default help

### Relevant log output

```
$ viash 
Unexpected error occurred! If you think this is a bug, please post
create an issue at https://github.com/viash-io/viash/issues containing
a reproducible example and the stack trace below.

viash - 0.7.1
Stacktrace:
java.util.NoSuchElementException: last of empty list
	at scala.collection.immutable.Nil$.last(List.scala:665)
	at scala.collection.immutable.Nil$.last(List.scala:661)
	at io.viash.Main$.mainCLI(Main.scala:178)
	at io.viash.Main$.mainCLIOrVersioned(Main.scala:117)
	at io.viash.Main$.main(Main.scala:64)
	at io.viash.Main.main(Main.scala)
```

### Version

OS: Ubuntu 22.04.3 LTS, Java: openjdk version "11.0.21" 2023-10-17, viash: 0.7.1

### Confirmation

- [X] I have searched the existing issues to make sure this is not a duplicate.
- [X] I have provided clear and concise information about the bug.

**Comment by mberacochea** [comment]
BTW, my viash installation is working
```
viash --help   
viash 0.7.1 (c) 2020 Data Intuitive

viash is a spec and a tool for defining execution contexts and converting execution instructions to concrete instantiations.

This program comes with ABSOLUTELY NO WARRANTY. This is free software, and you are welcome to redistribute it under certain conditions. For more information, see our license at the link below.
  https://github.com/viash-io/viash/blob/master/LICENSE.md

Usage:
  viash run config.vsh.yaml -- [arguments for script]
  viash build config.vsh.yaml
  viash test config.vsh.yaml
  viash ns build
  viash ns test
  viash ns list
  viash config view config.vsh.yaml
  viash config inject config.vsh.yaml

Check the help of a subcommand for more information, or the API available at:
  https://viash.io

Arguments:
  -h, --help      Show help message
  -v, --version   Show version of this program

Subcommands:
  run
  build
  test
  ns
  config
```

**Comment by rcannood** [comment]
Thanks @mberacochea. It appears this bug snuck in the release of Viash 0.6.4 as part of PR #294 :sweat_smile: I'm going to let the CI run to see whether #618 fixes the problem.

**Comment by rcannood** [comment]
#618 has been merged into the 0.8.x develop branch and will be included in release 0.8.3.

Thanks @mberacochea for reporting this!
